app/models/user.rb

```diff
@@ -7,6 +7,7 @@ class User < ActiveRecord::Base
   has_one :address, as: :addressable, dependent: :destroy
   has_many :auth_tokens, dependent: :destroy
   has_many :offers, foreign_key: :created_by_id, inverse_of: :created_by
+  has_many :orders, foreign_key: :created_by_id, inverse_of: :created_by
```

Comment:
has_many :created_orders, -> { order "id DESC" }, class_name: "Order", foreign_key: :created_by_id

We do have this. Still, we want to add orders association?